src/pipx/shared_libs.py

```diff
@@ -114,7 +114,7 @@ def upgrade(self, *, pip_args: List[str], verbose: bool = False, raises: bool =
             return
 
         if pip_args is None:
-            pip_args = []
+            pip_args = []  # type: ignore[unreachable]
```

Comment:
`pip_args` is not `Optional`, so in theory it cannot be `None`.

Do we still want to defend against an actual `None`?